src/components/SetupAssistant/SetupAssistantItem.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { ProgressRing } from '../Progress';
+
+const SetupAssistantItem = ({
+  children,
+  isExpanded,
+  title,
+  stepProgress,
+  step,
+  renderProgress,
+  renderExpanded,
+  renderAddon,
+  onExpand
+}, { assetBasePath }) => (
+  <li className="slds-setup-assistant__item">
+    <article className="slds-setup-assistant__step">
+      <div className={cx('slds-summary-detail', { 'slds-is-open': isExpanded })}>
+        {onExpand && (
+          <button
```

Comment:
I'm pretty sure that you could use an `<IconButton />` here, which would also alleviate the need for passing `contextTypes`. See the `With Custom ButtonIcon` example

Reply:
Wouldn't it be nicer to add a prop `iconClassName` to `IconButton` and pass that through?

Reply:
Hm I'm not sure about that. This is a fringe use case that should only be encountered when implementing other LDS components and I don't think we should litter `IconButton` with a prop in that case